lib/loaders/less.js

```diff
@@ -15,19 +15,19 @@ const applyOptionsCallback = require('../utils/apply-options-callback');
 
 /**
  * @param {WebpackConfig} webpackConfig
- * @param {bool} ignorePostCssLoader If true, postcss-loader will never be added
+ * @param {bool} useCssModules
  * @return {Array} of loaders to use for Less files
  */
 module.exports = {
-    getLoaders(webpackConfig, ignorePostCssLoader = false) {
```

Comment:
`ignorePostCssLoader` wasn't actually useful since `cssLoader.getLoaders()` look into the `webpackConfig` object for that info.